src/parser.rs

```
@@ -1314,11 +1333,24 @@ impl Parser {
         let span_start = self.position();
         self.keyword(b"while");
 
+        if self.is_operator() {
+            // TODO: flag parsing
+            self.next();
```

Maybe throw an error here with a WIP note instead of silently ignoring flags.